packages/e2e/lib/e2e_driver.dart

```diff
@@ -4,15 +4,59 @@ import 'dart:io';
 import 'package:e2e/common.dart' as e2e;
 import 'package:flutter_driver/flutter_driver.dart';
 
-Future<void> main() async {
+Future<void> main() async => e2eDriver();
+
+/// Adaptor to run E2E test using `flutter drive`.
+///
+/// `timeout` controls the longest time waited before the test ends.
+/// It is not necessarily the execution time for the test app.
+/// The test can and usually runs for shorter time if the test ends early.
```

Comment:
Maybe just `The test may finish sooner than the timeout`?  I don't think we should say anything about usual.